lib/dms.js

```diff
@@ -9,7 +9,7 @@ const logger = require('../utils/logger')
 class DmsModel {
   constructor(config) {
     this.config = config
-    this.api = config.get('API_URL')
+    this.api = config.get('INTERNAL_API_URL') || config.get('API_URL')
```

Comment:
So just to reflect your comment

> There are other places where frontend does connect to CKAN API (e.g., in custom themes developers don't always use this DMS library but they write custom code). But this DMS module makes 90% of requests, it also does a lot of concurrent requests where a package has lots of resources.


As I understand you still need API_URL to make regular request in some places. And that's cause custom themes are may also make request to CKAN_API, but why than needs to be done through NGINX, Is there a reason custom theme to not hit CKAN_API through internal network?

Reply:
Even if it does need to connect, here you get **one** of them, not both of them. Why can't we simply set API_URL to internal url and use that variable. Smth like. 
```
# If we need internal
API_URL=http://ckan-svc:5000

# If we need regular request
API_URL=https://data.ckan.datopian.com
```

Reply:
Just moving discussion from gchat here as well. If calls are made to ckan API directly from the browser (eg for loading views - tables graphs). than it needs to hit public ckan API. That can't go through the internal network. In that case we might need INTERNAL_API_URL for internal calls and API_URL for browser calls. But in that case we need **both**, not one of them.

Reply:
@zelima `API_URL` env var is used everywhere (both from browser and from node) and this is why we need a new explicit env var called `INTERNAL_API_URL`. You can see that it will be used only in the `dms` module which is responsible for all requests to CKAN API in the node app. The browser (client side JS) should still be able to make API calls to CKAN API as it is necessary for some apps and pages. But the number of those calls wouldn't be significant I think.